examples/2d/contributors.rs

```diff
@@ -52,19 +60,328 @@ const ALPHA: f32 = 0.92;
 
 const SHOWCASE_TIMER_SECS: f32 = 3.0;
 
-fn setup(
+const CONTRIBUTORS_LIST: &[&str] = &[
```

Comment:
This is probably too long (and either arbitrary or must be maintained). I would probably do something like `&["Carter Anderson", "And Many More"]` to make sure this stays usable as a learning example <3

Reply:
Reduced this arrray